ui/slider.js

```diff
@@ -126,21 +128,19 @@ return $.widget( "ui.slider", $.ui.mouse, {
 				this.range = $( "<div></div>" )
 					.appendTo( this.element );
 
-				classes = "ui-slider-range" +
-				// note: this isn't the most fittingly semantic framework class for this element,
-				// but worked best visually with a variety of themes
-				" ui-widget-header ui-corner-all";
+				this._addClass( this.range, "ui-slider-range", "ui-widget-header" );
```

Comment:
Considering the (delete) comment, we should consider moving "ui-widget-header" to the classes option default, along with that comment. Or keep the code as-is but leave the comment here, to be considered for the rewrite.